### PR TITLE
feat(intezer): add latest report command

### PR DESCRIPTION
**Pull request by davidt99**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
Add get the latest available report of a file instead of analyzing it)

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [x] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by davidt99** [comment]
@Itay4 Any chance I can get the exact error from circleci?

**Comment by Itay4** [comment]
> @Itay4 Any chance I can get the exact error from circleci?

You can follow [the link](https://app.circleci.com/pipelines/github/demisto/content/83014/workflows/ad54d51f-a6fc-4190-a304-18e96e60223f/jobs/342904) from [the comment](https://github.com/demisto/content/pull/12486#issuecomment-830819615) to CircleCI to see the errors

Also, please make sure to sign the CLA and fill the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA) so we can the review process

**Comment by davidt99** [comment]
I'm missing the partner id in the CLA but I'm working on that.
Unfortunately, I can't sign-up to CircleCI without giving it access to my private repositories so I can't do that. Any option to make the builds public?

**Comment by Itay4** [comment]
> I'm missing the partner id in the CLA but I'm working on that.
> Unfortunately, I can't sign-up to CircleCI without giving it access to my private repositories so I can't do that. Any option to make the builds public?

it it public
you can set grant access to public only according to https://discuss.circleci.com/t/github-org-permissions-only-access-public-repos/30525/2

**Comment by davidt99** [comment]
@Itay4 can you help me figure out what's wrong? when I run `demisto-sdk update-release-notes -i Packs/Intezer/` it yields "Either no changes were found in Intezer pack or the changes found should not be documented in the release notes file If relevant changes were made, please commit the changes and rerun the command". I'm on my master branch with the changes above.

**Comment by davidt99** [comment]
@Itay4 I fixed the tests and signed the CLA, we are good to go with the review.

File: Packs/Intezer/Integrations/IntezerV2/README.md [hunk 1]
```diff
@@ -20,6 +20,7 @@
 <p>You can execute these commands from the Demisto CLI, as part of an automation, or in a playbook. After you successfully execute a command, a DBot message appears in the War Room with the command details.</p>
 <ol>
 <li><a href="#h_6feb063f-66b1-45e5-89f0-1b6ff5142114" target="_self">Check file reputation by hash: intezer-analyze-by-hash</a></li>
+<li><a href="#h_353b4e58-45be-474d-b171-b8b803d216d0" target="_self">Check file reputation by latest report: intezer-get-analysis-result</a></li>
```
Comment (Itay4) [inline Packs/Intezer/Integrations/IntezerV2/README.md:23]:
```suggestion
<li><a href="#h_353b4e58-45be-474d-b171-b8b803d216d0" target="_self">Check file reputation by latest report: intezer-get-latest-report</a></li>
```

File: Packs/Intezer/Integrations/IntezerV2/IntezerV2.yml [hunk 2]
```diff
@@ -41,6 +41,39 @@ script:
...
       type: string
+  - arguments:
+    - default: false
+      description: Hash of the file to query. Supports SHA256, MD5 and SHA1
+      isArray: false
+      name: file_hash
+      required: true
+      secret: false
+    deprecated: false
+    description: Checks file reputation of the given hash, supports SHA256, SHA1 and MD5 by looking at the latest avaialble report
```
Comment (Itay4) [inline Packs/Intezer/Integrations/IntezerV2/IntezerV2.yml:52]:
```suggestion
    description: Checks file reputation of the given hash, supports SHA256, SHA1 and MD5 by looking at the latest available report
```

**Review by Itay4 — COMMENTED** [review]
looks good - added 2 minor comments

File: Packs/Intezer/Integrations/IntezerV2/README.md (hunk 1 above)
Comment (davidt99) [reply Packs/Intezer/Integrations/IntezerV2/README.md:23]: Thanks for picking this up! Is there a generator for this file? although the extension is markdown, the file is HTML and I edited it manually

**Review by Itay4 — APPROVED** [review]

**Pull request merged**